OWNERS

```diff
@@ -1,19 +1,13 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 reviewers:
-  - jayunit100
-  - hoegaarden
-  - andyxning
-  - neolit123
   - pohly
```

Comment:
shouldn't more reviewers from sig-instrumentation be added?

Reply:
They will need to go through the standard contributor ladder process. i don't think we've identified anyone that has been active in reviewing that should be listed as a reviewer but hasn't been.

Reply:
I think we can label the directory with wg/structured-logging and sig/instrumentation.